pages/_app.tsx

```diff
@@ -1,10 +1,10 @@
+import { ApolloProvider } from '@apollo/client'
 import React, { useEffect } from 'react'
 // TODO: type the posthog library
 // @ts-ignore
 import posthog from 'posthog-js'
-import { ApolloProvider } from '@apollo/react-hooks'
 import withApollo from '../helpers/withApollo'
-import { ApolloClient, NormalizedCacheObject } from 'apollo-boost'
+import { ApolloClient, NormalizedCacheObject } from '@apollo/client'
```

Comment:
can we move `ApolloProvider, ApolloClient, and NormalizedCacheObject` to be imported from the same line?